dataprofiler/profilers/utils.py

```diff
@@ -483,3 +483,23 @@ def find_diff_of_dicts(dict1, dict2):
         diff = "unchanged"
 
     return diff
+
+def get_capacity(data, unit='M'):
+    """
+    Get size (capacity) of the input data
+
+    :param data: list or array of data
+    :type data: Union[list, numpy.array, pandas.DataFrame]
+    :param unit: capacity unit (B, K, M, or G)
+    :type unit: string
+    :return: capacity of the input data
```

Comment:
need to update docstring to get rid of capacity

Reply:
changed